README.md

```diff
@@ -30,7 +30,8 @@ Read more about what `corobo` could do for you
 
 ## Environment Variables
 
-1. `COBOT_ROOT` - absolute path of the project root.
+1. `COBOT_ROOT` - absolute path of the project root. - Will soon be deprecated.
+1. `BOT_ROOT` - absolute path of the project root.
```

Comment:
1. The numbering is wrong ;)
2. "Will soon be depracated" - your commit already deprecates it, so it is deprecated right now :)
3. I would suggest you combine those to
   ```
   `BOT_ROOT` - absolute path of the project root. (Previously `COBOT_ROOT`, which is now deprecated)
   ```
   or similar phrasing :)